search-server/spacewalk-doc-indexes/spacewalk-doc-indexes.changes

```diff
@@ -1,3 +1,5 @@
+- Enable aarch64 builds
+- Bump version to 4.1.0 (bsc#1154940)
```

Comment:
Why is this changelog part of this PR or is it only because we forgot to add this entry in the past.

Reply:
Ah, right. In fact I think we should drop this package, right @mcalmer? I didn't see it at the list of packages from `rel-eng`.

Reply:
Talked to Michael, and yes, as I suspected, we don't really use this package anymore. I will rollback changes for it, and merge the PR.